training/data_preparation/training_requirements.txt

```diff
@@ -0,0 +1 @@
+# TODO  Add packages that the model training code requires when it is executed by WML
```

Comment:
Are there some packages missing? Else, we can remove the `TODO`

Reply:
Nothing missing as WML already has TF installed which is all that is required.